<a href="https://colab.research.google.com/github/sls3542/Project2/blob/main/DataSets_for_Project2ipynb.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import pandas as pd

In [2]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [4]:
path = "/content/drive/MyDrive/Intermediate Machine Learning/LoanPrediciton.csv"
df = pd.read_csv(path)
df.head()

Loan_ID Gender Married  Dependents     Education Self_Employed  \
0  LP001002   Male      No         0.0      Graduate            No   
1  LP001003   Male     Yes         1.0      Graduate            No   
2  LP001005   Male     Yes         0.0      Graduate           Yes   
3  LP001006   Male     Yes         0.0  Not Graduate            No   
4  LP001008   Male      No         0.0      Graduate            No   

   ApplicantIncome  CoapplicantIncome  LoanAmount  Loan_Amount_Term  \
0             5849                0.0         NaN             360.0   
1             4583             1508.0       128.0             360.0   
2             3000                0.0        66.0             360.0   
3             2583             2358.0       120.0             360.0   
4             6000                0.0       141.0             360.0   

   Credit_History Property_Area Loan_Status  
0             1.0         Urban           Y  
1             1.0         Rural           N  
2             1.0         Urban           Y  
3             1.0         Urban           Y  
4             1.0         Urban           Y

In [5]:
df.shape

(614, 13)

This data set comes from Kaggle. It is a collection of loan applicants, Credit History Amounts, Terms and Income Data. The target is to predict if the appliocant will be approved or not. Each row represents a single applicant. This is a classification project. There are 614 applicants in the dataset and 13 columns or features. the data has some missing and outlier values that must be cleaned. I anticpate it will take a fair amount of tunning to get the dataset to work given the quality of the data. I like this one because fintech is something I am interested in.

In [7]:
path = "/content/drive/MyDrive/Intermediate Machine Learning/salaries.csv"
df = pd.read_csv(path)
df.head()

work_year experience_level employment_type               job_title  salary  \
0       2022               SE              FT  Marketing Data Analyst  200000   
1       2022               EN              FT          Data Scientist   96000   
2       2022               SE              FT       Data Science Lead  165000   
3       2022               EN              FT          Data Scientist   27000   
4       2022               SE              FT           Data Engineer  185900   

  salary_currency  salary_in_usd employee_residence  remote_ratio  \
0             USD         200000                 GB           100   
1             CAD          74378                 CA           100   
2             USD         165000                 US            50   
3             GBP          33599                 GB            50   
4             USD         185900                 US             0   

  company_location company_size  
0               GB            S  
1               CA            L  
2               US            S  
3               GB            L  
4               US            M

In [8]:
df.shape

(1195, 11)

This data set comes from Kaggle. It is a collection of Data Science and Machine Learning employees and their salaries. The target is to predict the salary of the employee. Each row represents a single employee. This is a regression project. There are 1195 employees in the dataset and 11 columns or features. the data looks fairly clean but of course I will have to review it. This one is very relevant and contains a lot of different factors that can be explored individually. I really like this one.